pre_commit/commands/install_uninstall.py

```diff
@@ -37,7 +37,8 @@ def is_previous_pre_commit(filename):
     return any(hash in contents for hash in PREVIOUS_IDENTIFYING_HASHES)
 
 
-def install(runner, overwrite=False, hooks=False, hook_type='pre-commit'):
+def install(runner, overwrite=False, hooks=False, hook_type='pre-commit',
+            skip_on_missing_conf=False):
```

Comment:
p3: our [style guidelines](https://github.com/pre-commit/pre-commit/blob/41dcaff3fb53fb7819a1d783d67a9ccb42464c1d/CONTRIBUTING.md#multi-line-method-invocation) would suggest one of these instead:

```python
def install(
        runner, overwrite=False, hooks=False, hook_type='pre-commit',
        skip_on_missing_conf=False,
):
```

or 

```python
def install(
        runner,
        overwrite=False,
        ...
):
```

The reasoning here is:
- Prevents diff churn for adding / removing arguments
- Prevents merge conflicts (sometimes)
- Prevents arbitrary indentation that's awkward to do in editors